<launch>
	<node pkg="bobac2_audio" type="voice_nav_node" name="voice_nav" output="screen"/>
</launch>
